app/models/answer.rb

```diff
@@ -32,4 +35,23 @@ def accept
   def sort_value
     accepted ? Float::INFINITY : votes_count
   end
+
+  def create_action_verb
+    "Answered a Question"
+  end
+
+  def update_action_verb
+    "Updated his/her Answer on Question"
+  end
+
+  def should_create_activity?
+    true unless changed.include?('accepted')
+  end
+
+  def zhishi_url_options
+    {
+      question_id: question_id,
+      id: id,
```

Comment:
true although there is an ongoing conversation on whether it's a good practice since it's largely supported in the Python community